platform/qt/src/qmapboxgl.cpp

```diff
@@ -366,6 +366,54 @@ void QMapboxGLSettings::setAccessToken(const QString &token)
 */
 
 /*!
+    \enum QMapboxGL::MapChange
+
+    This enum represents the last changed occurred on the map state.
+
+    \value MapChangeRegionWillChange                      A region of the map will change, like
+    when resizing the map.
+
+    \value MapChangeRegionWillChangeAnimated              Not in use by QMapboxGL.
+
+    \value MapChangeRegionIsChanging                      A region of the map is changing.
+
+    \value MapChangeRegionDidChange                       A region of the map finished changing.
+
+    \value MapChangeRegionDidChangeAnimated               Not in use by QMapboxGL.
+
+    \value MapChangeWillStartLoadingMap                   The map is getting loaded. This state
+    is set only once right after QMapboxGL is created and a style is set.
```

Comment:
`qdoc` links automatically in this case.

https://tmpsantos.com.br/qmapboxgl/qmapboxgl.html#MapChange-enum